### DISCUSSION TITLE
I still have NOT obtained the benefit of education copilot after verification approved

### DISCUSSION THREAD

**Discussion by ppatrick21** [community/community#146031] [GitHub Education]
### Hi everyone,

I still have NOT obtained the benefit of education copilot after verification approved. Does anyone have met up with this issue or who should I ask? Thank you very much.

**Comment by davevad93** [comment]
Hi @ppatrick21 ,

Don't worry, the waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @ppatrick21 was approved and your benefits are now available.`

For further reference you can look at these discussions: 
#132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268, #140287, #145640

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)

**Reply by fornalewicz** [reply]
hi, my status has changed to purple and I also have no access to benefits... what can i do?

**Reply by davevad93** [reply]
Hey @fornalewicz  , if you still haven't access to your benefits, check if you can redeem them by going to the [pack offers page](https://education.github.com/pack), scroll down to the "***All offers***" section and select any offer, for example [Copilot](https://education.github.com/pack/redeem/copilot-student), if you can redeem it you are fine, if you can't, I suggest you to raise a ticket [here](https://support.github.com/contact/education), select "`I am a verified student having trouble redeeming my Global Campus offers`" and explain your issue. 

The same has been solved in this discussion #145995
